metadata-ingestion-examples/mce-cli/example-bootstrap.json

```diff
@@ -514,6 +534,68 @@
           ]
         }
       }
+    },
+    {
+      "proposedSnapshot": {
+        "com.linkedin.metadata.snapshot.GlossaryTermSnapshot": {
+          "urn": "urn:li:glossaryTerm:instruments.FinancialInstrument_v1",
+          "aspects": [
+            {
+              "com.linkedin.glossary.GlossaryTermInfo": {
+                "definition": "written contract that gives rise to both a financial asset of one entity and a financial liability of another entity",
+                "sourceRef": "EXTERNAL",
+                "termSource": "FIBO",
```

Comment:
Are the values for termSource and sourceRef switched up here?

Reply:
yeah, switched them back

Reply:
@shubham49 : a suggestion here that might keep things more streamlined. 

Can you use the [ExternalReference](https://github.com/linkedin/datahub/blob/master/metadata-models/src/main/pegasus/com/linkedin/common/ExternalReference.pdl) model to store the url for the source info.

Reply:
termSource and sourceRef are the attributes of my glossary term. We will show these fields on the UI glossary entity page.
We don't want to move these fields to `ExternalReference` because `ExternalReference` is used in other entities and above fields can only be attributed to glossary term.